Assets/Example/Scripts/CountlyEntryPoint.cs

```diff
@@ -38,7 +38,7 @@ private void Awake()
         countly = Countly.Instance;
     }
 
-    private void OnDisable()
+    private void OnDetroy()

```

Comment:
mistake

Reply:
it's not. listener will get remove on destroy not on disable. Disable could call many times, i.e when we disable game object etc...